examples/inference/image_to_image.py

```diff
@@ -135,6 +147,11 @@ def __call__(
         for i, t in tqdm(enumerate(self.scheduler.timesteps[t_start:])):
```

Comment:
Even though `i` is only used for LMS, I think it could be a bit easier to understand if we add a variable like:
```suggestion
        for i, t in tqdm(enumerate(self.scheduler.timesteps[t_start:])):
            timestep_index = t_start + i
```

Reply:
or just `t_index`

Reply:
And then just use this instead of `t_start + i` in the sigmas. IMO this could help a bit the readability here